dapps/profile.vue/src/i18n/de.ts

```diff
@@ -44,7 +100,49 @@ const i18n: any = {
       "desc": "Profilinformationen",
       "edit": "Profil bearbeiten"
     },
+    "device": {
+      "detail": {
+        "dataStreamSettings": {
+          "error": "Bitte fügen Sie Datenstrom-Einstellungsdateien ein!",
+          "label": "Datenstromeinstellungen *",
+          "placeholder": "Hinzufügen von Datenstrom-Dateien"
+        },
+        "location": {
+          "error": "Bitte geben Sie einen Standort an!",
+          "label": "Ort *",
+          "placeholder": "Geben Sie die Geräteplatzkoordinaten ein."
```

Comment:
```suggestion
          "placeholder": "Geben Sie die Standort Koordinaten ein."
```

Reply:
if we request coordinates we should offer at least in FE two input fields